crates/sui-types/src/messages.rs

```diff
@@ -107,13 +149,59 @@ pub struct MoveCall {
     pub arguments: Vec<CallArg>,
 }
 
+impl MoveCall {
+    pub fn validity_check(&self, config: &ProtocolConfig) -> SuiResult {
+        let is_blocked = BLOCKED_MOVE_FUNCTIONS.contains(&(
```

Comment:
should BLOCKED_MOVE_FUNCTIONS be in ProtocolConfig?

Reply:
I don't have a strong opinion other than, since this is no change from existing behavior, let's do it in a separate PR if we want